msteams-platform/messaging-extensions/build-bot-based-plugin.md

```diff
@@ -12,12 +12,13 @@ ms.date: 11/14/2023
 
 > [!NOTE]
 >
+> * Copilot for Microsoft 365 is in private preview. Ensure that you apply to the [Microsoft 365 Developer TAP](https://developer.microsoft.com/microsoft-365/tap) for a chance to receive early access to the first allocation of Copilot licenses.
```

Comment:
The way its phrased, this might sound contradictory with the 2nd statement in the note. How about something like:

Copilot for Microsoft 365 extensibility is in preview. Learn more about the test tenant and production tenant options for getting access to the [Copilot for Microsoft 365 dev environment](/microsoft-365-copilot/extensibility/prerequisites).

Reply:
Aw, now think we weren't reading thoroughly during the meeting. I am re-reading the requirements, and there are 2 options to get Copilot and one of them is to apply for TAP, so this doesn't seem mandatory. 

> Developers have two ways to obtain a developer environment for Copilot: (1) with a sandbox Microsoft 365 tenant with Copilot (available in limited preview through [TAP membership](https://learn.microsoft.com/en-us/microsoft-365-copilot/extensibility/prerequisites#isv-partners-in-microsoft-365-developer-tap)), or with (2) through an enterprise customer production environment with Microsoft Copilot for Microsoft 365 licenses.

I will need to mention this and edit the prerequisites on Copilot docs as well.

Reply:
updated.